Doc/reference/executionmodel.rst

```diff
@@ -330,6 +330,8 @@ enclosing namespace, but in the global namespace.  [#]_ The :func:`exec` and
 :func:`eval` functions have optional arguments to override the global and local
 namespace.  If only one namespace is specified, it is used for both.
 
+.. XXX(ncoghlan) above is only accurate for string execution. When executing code objects,
+   closure cells may now be passed explicitly to resolve co_freevars references.
```

Comment:
Yeah, I just added the comment because it was a can of worms I didn't want to open for this PR (it was true until the `closure` option was added to the builtins in 3.11, but has been slightly outdated since then).

https://github.com/python/cpython/issues/122826 filed

Reply:
Also added the issue reference to the comment.